src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- Please The App.js component should be class-based.
- After making the change, make sure that the calculator should render without problems